### ISSUE TITLE
Shortcut to open / hide preview

### ISSUE THREAD

**Issue by screendriver** [Glavin001/atom-preview#117]
Is there a keyboard shortcut to open and hide the preview?

**Comment by ddavison** [comment]
used to be Ctrl+Alt+Y I believe. you can set your own binding if you wanted. the action is `preview:toggle`

**Comment by Glavin001** [comment]
@ddavison is correct. See https://github.com/Glavin001/atom-preview/blob/master/lib/preview.coffee#L29-L34

**Comment by screendriver** [comment]
Ok, thank you. This should definitely be displayed when I press Command + Shift + P and search for `Preview` :wink: All other commands shows there their keyboard shortcuts.

**Comment by Glavin001** [comment]
I agree. Not sure why that is not the case. Could you inquire with the Atom team at https://github.com/atom/atom ? Thank you.

**Comment by screendriver** [comment]
I've made an issue for that: https://github.com/atom/atom/issues/9758

**Comment by ddavison** [comment]
is Ctrl+shift+Y really reserved though? in our keymap cson file, it's commented out.

**Comment by Glavin001** [comment]
Looks like it might be disabled / keymap not setup by default.

**Comment by Glavin001** [comment]
Well I am not sure if we want to reserve a keymapping or simple let Users create their own. I usually search for `preview` in the Atom command palette instead.

**Comment by screendriver** [comment]
For new users that are using preview it's a lot more comfortable to have a keyboard shortcut :wink:

**Comment by Glavin001** [comment]
Sure. I am open to suggestions for what the key mapping should be, and that does not collide with other packages.

**Comment by screendriver** [comment]
Command + Shift + P is unfortunately occupied :wink: 

What about `⌘P`  `⌘R` (**PR**eview)?

**Comment by Glavin001** [comment]
Is there a website to search for existing keyboard shortcuts across all packages for Atom? That would be helpful to make sure we avoid collisions.

`⌘P` + `⌘R` sounds good so far. Doubt anyone has taken that already.

I am ready and willing to receive any Pull Requests on this matter :smile:. Thanks!

**Comment by screendriver** [comment]
> Is there a website to search for existing keyboard shortcuts across all packages for Atom? That would be helpful to make sure we avoid collisions.

That would be a great feature. But I don't believe that something exist that can do that.
Sorry I can't make a pull request. I don't know how to write packages for Atom and how to modify `atom-preview` for that :confused:
